### ISSUE TITLE
Demo use of dachar and daops for fixing data

### ISSUE THREAD

**Issue by bouweandela** [ESMValGroup/ESMValCore#787]
It would be nice to demo how we could use the character/fixes defined by [dachar](https://github.com/roocs/dachar) to fix data with [daops](https://github.com/roocs/daops) and then [convert that from an xarray to iris cube](http://xarray.pydata.org/en/stable/generated/xarray.DataArray.to_iris.html) for use in the ESMValCore.

Example fixes:

Fix standard name:
https://github.com/ESMValGroup/ESMValCore/blob/ea6618c3fd014d32f29931853cc16c22453c4e09/esmvalcore/cmor/_fixes/cmip5/hadgem2_cc.py#L34-L55

Fix fill value:
https://github.com/ESMValGroup/ESMValCore/blob/ea6618c3fd014d32f29931853cc16c22453c4e09/esmvalcore/cmor/_fixes/cmip5/cesm1_bgc.py#L8-L26

Fix hybrid sigma coordinate (more complicated, only if time permits):
https://github.com/ESMValGroup/ESMValCore/blob/ea6618c3fd014d32f29931853cc16c22453c4e09/esmvalcore/cmor/_fixes/cmip6/cesm2.py#L11-L76

The plan is that:
- @agstephens or one of his team members will prepare the character and fix .json files with `dachar` and possibly implement the fixes mentioned above in `daops`.
- Someone from @ESMValGroup/esmvaltool-coreteam (@jvegasbsc would you have time?) Will make a demo how this could be used to load the fixed data into the ESMValCore.

Related to #755

**Comment by jvegasbsc** [comment]
> @jvegasbsc would you have time?

Yes, I can do it. It could save me lot's of time, so it is a good investment

**Comment by valeriupredoi** [comment]
way to go, Javi! :beer:

**Comment by agstephens** [comment]
@ellesmith88, tagging you on this issue as a collaboration point with ESMValTool.

**Comment by ellesmith88** [comment]
Hi @zklaus @nielsdrost @bouweandela @jvegasbsc @valeriupredoi and @agstephens I've been working on this on our end and have two PRs open with the `json` files and fix classes in `dachar` and the fixes implemented in `daops`: https://github.com/roocs/daops/pull/40 and https://github.com/roocs/dachar/pull/88. 

This issue (https://github.com/roocs/daops/issues/38) and my step by step comment should explain what I've done - I have looked at all fixes now though. Hopefully that makes sense but let me know if it doesn't. It would be great to get some feedback and see what you think.

**Comment by bouweandela** [comment]
We discussed this recently at an IS-ENES3 meeting with @agstephens @cehbrecht @nielsdrost @jvegasbsc @remi-kazeroni and @valeriupredoi. The conclusion was that we are still very interested in doing this and think it's a great idea, but @jvegasbsc indicated that he does not have time to work on this at the moment and no-one else volunteered either. We may need to find new funding to work on this further.
